gradle/java-compiler-settings.gradle

```diff
@@ -35,6 +35,49 @@ task javadocJar(type: Jar, dependsOn: javadoc) {
 	from javadoc.destinationDir
 }
 
+ext.signMethod = { jarfile -> 
+	println "Signing $jarfile"
+	def SIGNING_SERVICE = 'http://build.eclipse.org:31338/sign'
+	def STDOUT_FORMAT = '    %{size_upload} bytes uploaded, %{size_download} bytes downloaded (%{time_total} s)\\n'
+
+	ProcessBuilder curl_pb = new ProcessBuilder("curl", 
+		"--fail", "--silent", "--show-error", "--output",
```

Comment:
we had to add some retrys to Xtext here as the service is instable sometimes

Reply:
https://git.eclipse.org/c/cbi/org.eclipse.cbi.git/tree/maven-plugins/eclipse-jarsigner-plugin/src/main/java/org/eclipse/cbi/maven/plugins/jarsigner/mojo/SignMojo.java does 3 retrys too

Reply:
Thanks, I had seen some retries in other code - I can look at that once we have some instability - at the moment the full build time is very short so running it again is less time consuming. I assume xtext has a much longer build time, so having a signing failure would be much more inconvenient?

Reply:
indeed